modules/nf-core/gatk4/applybqsr/main.nf

```diff
@@ -35,7 +35,7 @@ process GATK4_APPLYBQSR {
     """
     gatk --java-options "-Xmx${avail_mem}g" ApplyBQSR \\
         --input $input \\
-        --output ${prefix} \\
+        --output ${prefix}.${input.getExtension()} \\
```

Comment:
Shouldn't that be fixed in nf-core/modules instead?

Reply:
sure, but this is not part of this PR?